data/operators/telecom/exchange.json

```diff
@@ -180,6 +180,19 @@
         "telecom": "exchange"
       }
     },
+    {
+      "displayName": "Glo",
+      "locationSet": {
+        "include": ["ng", "bj"],
+      },
+      "tags": {
+        "name": "Glo",
+        "operator": "Globacom",
+        "operator:wikidata": "Q66365656",
```

Comment:
Those Wikidata and Wikipedia don't match

[Q66365656 (Glo (company))](https://www.wikidata.org/wiki/Q66365656) don't relevent to any Wikipedia pages, and I can't find a page named with "Globacom Limited" in English Wikipedia.

I only can found a [Glo (company)](https://en.wikipedia.org/wiki/Glo_(company)), and its related Wikidata is [Q5570033 (Globacom)](https://www.wikidata.org/wiki/Q5570033)

Reply:
Sure. Noted